src/components/Menu-Badge/MenuBadge.tsx

```diff
@@ -0,0 +1,100 @@
+import { Box, Typography } from '@mui/material';
+
+import { COLOR } from './MenuBadge.constraints';
+
+type MenuBadgeProps = {
```

Comment:
You define a local `type MenuBadgeProps` inside the component file, while `src/components/Menu-Badge/MenuBadge.types.ts` also exports an interface with the same name. This duplication can lead to drift and confusion; you should import and use the centralized type.

**Sugerencia:**
```typescript
Remove the local `MenuBadgeProps` type and import the interface from `MenuBadge.types.ts`. Also ensure the exported type name follows the project pattern `MenuBadgeProps` (which it does). Example:

import type { MenuBadgeProps } from './MenuBadge.types';

export const MenuBadge = ({ label, variant = 'default' }: MenuBadgeProps) => { ... }
```